tests/test_openai_gym.py

```diff
@@ -0,0 +1,189 @@
+#
+# License: See LICENSE.md file
+# GitHub: https://github.com/Baekalfen/PyBoy
+#
+
+import numpy as np
+import pytest
+from pyboy import PyBoy, WindowEvent
+from pyboy.botsupport.constants import COLS, ROWS
+
+try:
+    from .utils import tetris_rom
+except:
+    tetris_rom = "ROMs/Tetris.GB"
```

Comment:
The wrong Tetris ROM could have been caught here. `utils.tetris_rom` is checksum based, and doesn't recognize your ROM as the correct Tetris ROM